deploy-agent/deployd/common/config.py

```diff
@@ -14,14 +15,16 @@
 
 import logging
 import os
-from ConfigParser import SafeConfigParser
 
 from deployd.common.exceptions import DeployConfigException
 from deployd.common.types import DeployType
 from deployd.common.utils import exit_abruptly
 from deployd.types.deploy_stage import DeployStage
 from deployd.types.opcode import OperationCode
 
+from configparser import ConfigParser
```

Comment:
just double check it is also supported in py2 and it is a safe.

Reply:
yup confirmed by running unit tests work on PY2 and PY3